tox.ini

```diff
@@ -8,8 +8,8 @@ deps =
   ansible-lint
   flake8
   molecule
-  molecule-plugins[docker]
-  molecule-plugins[vagrant]
+  molecule-containers
+  molecule-vagrant
```

Comment:
Why did you revert this change? Using `molecule-containers` is the old naming scheme